### ISSUE TITLE
Error while starting recording

### ISSUE THREAD

**Issue by zohaib09** [MetaProvide/talked#48]
Hi

I have installed talked on ubuntu, and intergrate it with nextcloud talk. When i run command "/recording start" i get following error

Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/talked/talked/lib/python3.8/site-packages/talked/recorder.py", line 52, in start
    browser = launch_browser(call_link, msg_queue, nextcloud_version, grid_view)
  File "/opt/talked/talked/lib/python3.8/site-packages/talked/recorder.py", line 132, in launch_browser
    change_name_of_user(driver, nextcloud_version)
  File "/opt/talked/talked/lib/python3.8/site-packages/talked/recorder.py", line 208, in change_name_of_user
    driver.find_element_by_css_selector("input.username-form__input").send_keys(
AttributeError: 'WebDriver' object has no attribute 'find_element_by_css_selector'

talked: 0.4V
Nextcloud: 24

Please suggest solution

**Comment by zohaib09** [comment]
Please help.

I had setup same talked server with nextcloud version 23, talk 0.3.2 that was working perfectly. We have upgraded to nextcloud 24. At same time we have reinstalled talked with new version 0.4 on new server (ubuntu) followed same procedure, but this time facing error as mentioned in previous post.

**Comment by zohaib09** [comment]
Issue has been resolved. Some of the syntax is not readable by selenium 4.3 in recorder.py

Issue description: When we installed talked, dependencies were installed automatically, One dependency name selenium was installed with newer version 4.3. 

In record.py there are number of syntax that are not able to read by selenium version 4.3. Developer needs to update syntax in recorder.py then it will be supported by newer version of selenium

Workaround: We had downgraded version of selenium from 4.3 to 4.1.3 now syntax is readable by selenium in recorder.py
